questions.md

```diff
@@ -0,0 +1,6 @@
+# Few more questions on Pull request
+
+* Is it possible to accept a pull request into a new branch?
+* Is a pull request the same as git pull?
+* How do I update a GitHub forked repository?
```

Comment:
Once you have forked a repository you can manipulate it as you would with a personal project.